exercise_dirty.py

```diff
@@ -0,0 +1,30 @@
+foo = []
+
+from .models import Expense
+from collections import namedtuple
+
+MyExpense = namedtuple('F', ['type_', 'amount'])
+
+# test data
+foo.append(MyExpense('food', 4))
+foo.append(MyExpense('food', 3))
+foo.append(MyExpense('var', 3))
+foo.append(MyExpense('dog', 1))
+
+
+def summarizeExpenses(min_amount, input):
```

Comment:
I think the parameters orders could be switched. 
I think also the min_amount could have default value e.g.  min_amount=1

Not sure about the project style  but also typing could be added; 
e.g.:(min_amount: int, input: list) -> None: